src/masternode-payments.cpp

```diff
@@ -785,6 +785,40 @@ void CMasternodePayments::ProcessBlock(int nBlockHeight)
     nLastBlockHeight = nBlockHeight;
 }
 
+bool CMasternodePayments::CreateMNWinnerPayment(const CTxIn& mnVinWinner, int paymentBlockHeight, const CScript& payeeScript)
+{
+    if (!fMasterNode) {
+        throw std::runtime_error("Error trying to create a MN payment without being a MN");
+    }
+
+    // sign it with the active masternode keys
+    CPubKey pubKeyMasternode; CKey keyMasternode;
+    activeMasternode.GetKeys(keyMasternode, pubKeyMasternode);
```

Comment:
this will not work for deterministic masternodes (and they will also create mnw messages during the transition phase). 
Should have used `GetActiveMasternodeKeys()`.
But... this is going to change with #2421, you might want to check that PR first.